### ISSUE TITLE
min_learning_rate in ReduceLROnPlateau scheduler

### ISSUE THREAD

**Issue by Adamits** [CUNY-CL/yoyodyne#145]
My labmate was using the `ReduceOnPlateau` scheduler and gets an error related tot he arg `min_learning_rate`. 

This is because we inherit from the [pytorch scheduler](https://pytorch.org/docs/1.13/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#torch.optim.lr_scheduler.ReduceLROnPlateau), which uses `min_lr`. It looks like we reverted this a couple months ago and changed `min_lr` to `minimum_lr` [here](https://github.com/CUNY-CL/yoyodyne/blob/master/yoyodyne/schedulers.py#L151). 

Reverting it back should fix the bug.
